modules/customization/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -0,0 +1,116 @@
+[id='proc-customize-rhdh-sidebar-menuitems_{context}']
+= Customizing the sidebar navigation menu items for your {product-short} instance
+
+The sidebar menu in {product} consists of two main parts:
+
+* *Main menu items*: These items are the static menu items that form the core navigation structure of sidebar. These menu items remain consistent and are predefined.
+
+* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The main menu items section is dynamic and can change based on your preferences and installed plugins.
+
+You can customize the sidebar menu items in your `app-config-rhdh.yaml` file. The order and parent-child relationships of sidebar main menu items can be customized using the `dynamicPlugins.frontend.default.main-menu-items.menuItems` field for the main menu items, or the `dynamicPlugins.frontend.<package_name>.menuItems` field for dynamic plugin menu items in the `app-config-rhdh.yaml` file:
```

Comment:
Consider applying the procedure template:

* Add the `.Procedure` subtitle.
* Add steps as an ordered list ((or a bullet point for signle-step procedures).

Reply:
Here we should mention that order and parent-child relationship are configurable within their respective scope, which means `priority` or `parent` can be defined and applied for each section, not mixed. Let me know if I didn't explain it well or you have included this somewhere else that I missed.

Reply:
Done